components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -2255,6 +2255,12 @@ public int compare(APIUsageDTO o1, APIUsageDTO o2) {
     @Override
     public void deployArtifacts(String url, String user, String pass) throws Exception {
 
+        if (url.trim().equals("")) {
```

Comment:
You may do this as mentioned above. Do we need to check url empty or not to avoid null pointer? May be StringUtils.isEmpty can be used